addons/l10n_sa_edi/tests/test_invoice.py

```diff
@@ -0,0 +1,32 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.addons.l10n_sa_edi.tests.common import TestSaEdiCommon
+from odoo.tests import tagged
+from odoo import Command
```

Comment:
order of imports, let's follow the guidelines: https://www.odoo.com/documentation/17.0/contributing/development/coding_guidelines.html#imports